ts/features/idpay/unsubscription/hooks/useConfirmationChecks.ts

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+type ConfirmationChecks = {
+  values: ReadonlyArray<boolean>;
+  toggle: (atIndex: number) => void;
+  areFulfilled: boolean;
+};
+
+/**
+ * A custom React Hook that manages an arbitrary array of boolean values and tells if all values are true or not.
+ * @param length - The length of the `values` array.
+ */
+export const useConfirmationChecks = (length: number): ConfirmationChecks => {
```

Comment:
Can we extract this?

Reply:
Moved into common folder. See https://github.com/pagopa/io-app/pull/4462/commits/936923a530056d230b1782d13115c3f7b650d16b